UPGRADE.md

```diff
@@ -8,6 +8,17 @@ does not have any particular instructions.
 
 ## Upcoming release
 
+### Universal
+
+### Changed default postgres driver to pgx
+- If you encounter any problems with the persistence layer please [submit an issue](https://github.com/kumahq/kuma/issues/new) and temporarily switch to the previous driver (`lib/pq`) by setting
```

Comment:
nit: should the `DriverName` be `pg`?

Reply:
no, its `pq` - https://github.com/lib/pq

Reply:
Right I meant pq

Reply:
Well the driver name is then passed to `db.Open` so we would have to map there. I can change it so that it's consistent. WDYT?

Reply:
Sorry I'm not clear here. I'm talking about the possible values of: `KUMA_STORE_POSTGRES_DRIVER_NAME` currently it is: `pgx` or `postgres`. Should it be `pgx` and `pq` instead?

Reply:
Even though the lib is called `lib/pq` the driver name is "postgres" - https://github.com/lib/pq/blob/922c00e176fb3960d912dc2c7f67ea2cf18d27b0/conn.go#L61 so I'm fine with calling it `pq` and when passing in anywhere mapping it to `postgres`. Or do you want to this:

```suggestion
- If you encounter any problems with the persistence layer please [submit an issue](https://github.com/kumahq/kuma/issues/new) and temporarily switch to the previous driver (`postgres`) by setting
```

?

Reply:
I'm going with current value to move this forward, if you want this change let me know I'll do it in the next PR.